wiki/History_of_osu!/2012/en.md

```diff
@@ -1,106 +1,138 @@
 # History of osu! 2012
 
+## March
+
+A new feature was tested out which added an integrated [IRC](https://en.wikipedia.org/wiki/IRC) web chat to the osu! website.[^web-chat]
```

Comment:
For this description, I am not sure if I interpreted the messages in the [respective forum thread](https://osu.ppy.sh/community/forums/topics/77845?n=1) correctly.